content/zh-cn/_index.html

```diff
@@ -22,7 +22,7 @@
 {{% blocks/feature image="scalable" %}}
 
 <!-- #### Planet Scale -->
-#### 星际尺度
+#### 全球规模的可扩展性
```

Comment:
The original has more advertising feeling on this home page

Reply:
`星际尺度` Can you describe what this means?

Reply:
I think this is a direct translation and does not convey the meaning

Reply:
If the translation takes literal translation as the standard, this PR can be turned off, and the advertising words should be easier to understand, rather than forcing a word out